src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -0,0 +1,219 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.CreateArticleModalComponentObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+/**
+ * Created by Rodriuki on 11/06/15.
+ * Created by nikodamn on 20/07/15
+ */
+public class PortableInfoboxPageObject extends WikiBasePageObject {
+
+  @FindBy(css = ".portable-infobox-image-wrapper")
+  private WebElement pInfoImage;
+  @FindBy(css = ".portable-infobox-title")
+  private WebElement pInfoTitle;
+  @FindBy(css = ".portable-infobox-header")
+  private List<WebElement> pInfoTitleH3;
+  @FindBy(css = ".portable-infobox .external")
+  private WebElement pInfoExternalLink;
+  @FindBy(css = ".portable-infobox-navigation a[href*='redlink']")
+  private List<WebElement> pInfoRedlLink;
+  @FindBy(css = "b")
+  private List<WebElement> boldElements;
+  @FindBy(css = "i")
+  private List<WebElement> italicElements;
+  @FindBy(css = ".portable-infobox a[href*='/wiki/']")
+  private List <WebElement> pInfoInternalLinks;
+  @FindBy(css = ".WikiaLightbox")
+  private WebElement lightbox;
+  @FindBy(css = ".portable-infobox")
+  private WebElement infoboxLayout;
+  @FindBy(css = ".tabberlive")
+  private WebElement tabber;
+  @FindBy(css = ".tabbertab .image")
+  private WebElement tabberImage;
+  @FindBy(css = ".portable-infobox-item-value ul li")
+  private List <WebElement> unorderedElementList;
+  @FindBy(css = ".portable-infobox-item-value ol li")
+  private List <WebElement> orderedElementList;
+  @FindBy(css = ".reference")
+  private WebElement referenceElements;
+  @FindBy(css = ".portable-infobox-item-label")
+  private WebElement h3Elements;
+  @FindBy(css = "#CreatePageModalDialog")
+  private WebElement createArticleModal;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "createArticleModal" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)